test/common/tls/test_data/san_multiple_othername_string_type_cert.cfg

```diff
@@ -0,0 +1,56 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = v3_req
+
+[req_distinguished_name]
+countryName = US
+countryName_default = US
+stateOrProvinceName = California
+stateOrProvinceName_default = California
+localityName = San Francisco
+localityName_default = San Francisco
+organizationName = Lyft
+organizationName_default = Lyft
+organizationalUnitName = Lyft Engineering
+organizationalUnitName_default = Lyft Engineering
+commonName = Test Server
+commonName_default = Test Server
+commonName_max  = 64
+
+[v3_req]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+subjectAltName = @alt_names
+subjectKeyIdentifier = hash
+
+[v3_ca]
+basicConstraints = critical, CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+subjectAltName = @alt_names
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always
+
+[alt_names]
+otherName.1 = 1.3.6.1.4.1.311.20.2.3;BITSTRING:01010101
+otherName.2 = 1.3.6.1.4.1.311.20.2.4;OCTETSTRING:48656C6C6F20576F726C64
+otherName.3 = 1.3.6.1.4.1.311.20.2.5;PRINTABLESTRING:PrintableStringExample
+otherName.4 = 1.3.6.1.4.1.311.20.2.6;T61STRING:T61StringExample
+otherName.5 = 1.3.6.1.4.1.311.20.2.7;IA5STRING:IA5StringExample
+otherName.6 = 1.3.6.1.4.1.311.20.2.8;GENERALSTRING:GeneralStringExample
+otherName.7 = 1.3.6.1.4.1.311.20.2.9;UNIVERSALSTRING:UniversalStringExample
+otherName.8 = 1.3.6.1.4.1.311.20.2.10;UTCTIME:230616120000Z
+otherName.9 = 1.3.6.1.4.1.311.20.2.11;GENERALIZEDTIME:20230616120000Z
+otherName.10 = 1.3.6.1.4.1.311.20.2.12;VISIBLESTRING:VisibleStringExample
+otherName.11 = 1.3.6.1.4.1.311.20.2.13;UTF8STRING:UTF8StringExample
+otherName.12 = 1.3.6.1.4.1.311.20.2.3;SET:test_set
+otherName.13 = 1.3.6.1.4.1.311.20.2.3;SEQUENCE:test_sequence
```

Comment:
These two use the same oid as BITSTRING above. Should they be separate oid's?

Reply:
I added them that way to validate the functionality that despite having same OIDs, our matcher should be able to match against all possible values with this OID. Happy to change if separate OIDs would be better

Reply:
Got it; that makes sense. Can you add a comment explaining that in this file?